### PR TITLE
api: add index refresh

### PR DESCRIPTION
**Pull request by ntarocco**
- closes https://github.com/inveniosoftware/invenio-communities/issues/620

### CODE REVIEW

File: invenio_indexer/api.py [hunk 1]
```diff
@@ -162,6 +162,14 @@ def index_by_id(self, record_uuid, **kwargs):
         """
         return self.index(self.record_cls.get_record(record_uuid), **kwargs)
 
+    def refresh(self, index, **kwargs):
+        """Refresh an index.
+
+        :param index: the index to refresh.
+        """
+        index_name = build_alias_name(index)
```
Comment (ppanero) [inline invenio_indexer/api.py:170]:
```suggestion
        index_name = self._prepare_index(index)
```

File: tests/test_api.py [hunk 2]
```diff
@@ -204,6 +204,17 @@ def test_index(app):
             assert fun.called
 
 
+def test_refresh(app):
+    """Test index refresh."""
+    with app.app_context():
+        client_mock = MagicMock()
+        ri = RecordIndexer(search_client=client_mock, version_type='force')
+        ri.refresh(app.config['INDEXER_DEFAULT_INDEX'])
```
Comment (ppanero) [inline tests/test_api.py:212]: In this test we are passing a string. However, from `IndexRefreshOp` we are passing a `Index` object.

**Review by ntarocco — COMMENTED** [review]
LGTM!

**Comment by ppanero** [comment]
@kpsherva tested:
- instance setup
- community creation
- metadata only record publishing
- metadata only record inclusion in community
- community member invite

I think we are good to go :)

**Review by kpsherva — APPROVED** [review]

**Pull request merged**